pom.xml

```diff
@@ -55,6 +55,11 @@
             <artifactId>springfox-swagger-ui</artifactId>
             <version>${swagger.version}</version>
         </dependency>
+        <dependency>
```

Comment:
Ugly naming for PR, in future name it correctly. Imagine that I dont know what is snippet what is docs and other.